CONTRIBUTING.md

```diff
@@ -4,10 +4,20 @@ For anyone who knows [Git](https://git-scm.com) and JavaScript basics: feel free
 
 ## Filing [issues](https://docs.github.com/en/github/managing-your-work-on-github/about-issues)
 
-Found a bug? Report it! Be sure to check if it is already reported by using the search bar [here](https://github.com/ScratchAddons/ScratchAddons/issues). If there are no similar issues, you can create a new one. We will take a look at it.
+Found a bug? Report it in an issue! But first, be sure to check if it was already reported by using the search bar [here](https://github.com/ScratchAddons/ScratchAddons/issues). If there are no similar issues, you can create a new one. We will take a look at it.
 
 Also, feel free to ask questions, suggest new features, and more.
 
 ## Creating [pull requests](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
 
-Contributions are welcome! Be sure to file an issue first (see above). If you know a fix for that issue, or you know a fix for a different one, you can create a pull request. Fork this repository, create a new branch from the 'master' branch, and make your changes on the new branch. Now, create a pull request on the origin repository (ScratchAddons/ScratchAddons). We will review your pull request.
+Contributions are welcome! If you have something in mind, it's best if you file an issue first (see above). That way, we can discuss it before you start a pull request. If you know a fix for that issue (or any other issue), you can follow these steps to create a pull request:
+1. [Fork](https://github.com/ScratchAddons/ScratchAddons/fork) this repository if you haven't already.
+2. [Sync your fork's `master` branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) to ensure it is up to date.
+3. [Create a new branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository) from the `master` branch.
+4. Make your changes and test them locally on a modified [download of Scratch Addons](https://github.com/ScratchAddons/ScratchAddons/archive/refs/heads/master.zip).
```

Comment:
I'm new to git, but it seems after I forked the repository and pulled, I had the files locally to modify. Why would someone download the .zip version at this point?

Reply:
No one who knows a little git does. This guide should really juat teach the basics of git instead (or even jsut suggest installing GitHub desktop) it makes things so much easier later on.

Reply:
IMO, don't explain git/GitHub related things that much, just add link to good tutorial (for example [this](https://docs.github.com/en/get-started/quickstart/hello-world)).

Reply:
That works. I just don't think we should be telling contributors to download the zip when they can just run git clone and have acsess go all of git's features.

Reply:
Is [this](https://docs.github.com/en/get-started/quickstart/contributing-to-projects) a good guide?

Reply:
I was almost done with my next edit, but I'm once again missing a link. I need to explain how to switch and commit to a new branch using Git. Where should I link?

Reply:
I used GitHub desktop, and I think it was a more gentle learning curve than command line git. Something like @DNin01's bullet points, together with [this getting started link](https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/overview/getting-started-with-github-desktop),  and maybe [this link about forking/cloning](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/adding-and-cloning-repositories/cloning-and-forking-repositories-from-github-desktop) (even though you can get there from the first link) would have been a good leg up when I got started.